SwrveConversationSDK/src/main/java/com/swrve/sdk/conversations/ui/ConversationRoundedLinearLayout.java

```diff
@@ -36,12 +48,25 @@ protected void onSizeChanged(int w, int h, int oldw, int oldh) {
         path.close();
     }
 
+    private boolean clippingNotSupported = false;
+
     @Override
     protected void dispatchDraw(Canvas canvas) {
-        int save = canvas.save();
-        canvas.clipPath(path);
-        super.dispatchDraw(canvas);
-        canvas.restoreToCount(save);
+        boolean doClip = (radius > 0 && !clippingNotSupported);
+        if (doClip) {
```

Comment:
I would leave this logic out. Once hardware acceleration is disabled, the clipPath method is supported down to api level 1.